pkg/vex/vex.go

```diff
@@ -229,24 +244,58 @@ func SortDocuments(docs []*VEX) []*VEX {
 
 // Open tries to autodetect the vex format and open it
 func Open(path string) (*VEX, error) {
+	logrus.Info("Abriendo")
```

Comment:
Just checking to see if this was intentional :)

Reply:
No, deleted :)